app/models/automation_request.rb

```diff
@@ -27,7 +27,7 @@ def self.create_from_ws(version, user, uri_parts, parameters, requester)
     options[:class_name]    = (options.delete(:class) || DEFAULT_CLASS).strip.gsub(/(^\/|\/$)/, "")
     options[:instance_name] = (options.delete(:instance) || DEFAULT_INSTANCE).strip
     options[:schedule_type] = parameters['schedule_time'].present? ? "schedule" : "immediately"
-    options[:schedule_time] = parameters['schedule_time'].to_i.days.from_now if parameters['schedule_time']
+    options[:schedule_time] = Time.parse(parameters['schedule_time']) if parameters['schedule_time']
```

Comment:
Was this in local or UTC time before?  And after this change, it depends what they provide in the string?  Do we need to check if it's a string?  Do we accept number of days as we did previously?